WindowsServerDocs/identity/ad-fs/overview/AD-FS-FAQ.md

```diff
@@ -93,7 +93,7 @@ The value of “sub” claim is the hash of client ID + anchor claim value.
 The lifetime of refresh token will be the lifetime of the token that ADFS got from remote claims provider trust. The lifetime of the access token will be the token lifetime of the relying party for which access token is being issued.
 
 ### I need to return profile and email scopes as well in addition to the OpenId scope. Can I obtain additional information using scopes? How to do it in AD FS?
-You can use customized id_token to add relevant information in the id_token itself. For more, read here.
+You can use customized id_token to add relevant information in the id_token itself. For more information, read [here](../development/Custom-Id-Tokens-in-AD-FS.md).
```

Comment:
While this change is technically right it does not meet our accessibility requirements. The link text "here" doesn't tell someone with a screen reader what they are clicking on. If you change the link text to something descriptive like 

For more information, see the article [Customize claims to be emitted in id_token]

I would be happy to approve the PR.